### DISCUSSION TITLE
Should judgement feedback on osu!mania flashlight in front of flashlight mask?

### DISCUSSION THREAD

**Discussion by IkaWaAyuMu** [ppy/osu#20664] [Ideas]
It's just quite frustrated that I don't know my progress during flashlight play. (Especially on Argon skin that bubble keep splashing at me :rofl:)
this should be an option: as bright object suddenly appear in the dark may cause some reading issue to some players.
![image](https://user-images.githubusercontent.com/7066150/194756496-f22af9fb-3fef-434c-bcd5-f9f1fddc94a1.png)
![image](https://user-images.githubusercontent.com/7066150/194756624-ce82edd0-075e-4122-b954-17daa826bf84.png)

*Stable also hide the judgement behind the mask. So, implement this cause it to not stable match.*

**Comment by smoogipoo** [comment]
I'd say yes. Not sure there's much of a point in hiding it.
